packages/web-react/src/components/Stack/Stack.tsx

```diff
@@ -14,11 +14,18 @@ const defaultProps: SpiritStackProps = {
 
 export const Stack = <T extends ElementType = 'div'>(props: SpiritStackProps<T>): JSX.Element => {
   const { elementType: ElementTag = 'div', children, ...restProps } = props;
-  const { classProps, props: modifiedProps } = useStackStyleProps(restProps);
+  const { classProps, props: modifiedProps, styleProps: stackStyle } = useStackStyleProps(restProps);
   const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
 
+  const stackStyleProps = {
+    style: {
+      ...styleProps.style,
+      ...stackStyle,
+    },
+  };
+
   return (
-    <ElementTag {...otherProps} {...styleProps} className={classNames(classProps, styleProps.className)}>
+    <ElementTag {...otherProps} {...stackStyleProps} className={classNames(classProps, styleProps.className)}>
```

Comment:
Maybe I would suggest merging the styles the same way we do it with the `className`, e. g. `<ElementTag style={{ ...styleProps.style, ...stackStyle }} />`. Because it is more consistent with the current way. What do you think?